cidemipyme-gatsby/src/data/send_email.php

```diff
@@ -0,0 +1,38 @@
+<?php
+if(isset($_POST["email"])) {
+  $to = "miraleeb@hotmail.com";
+  $subject = $_POST["subject"];
+  $from = $_POST["email"];
+  $message = $_POST["message"];
+  $html_message = '
```

Comment:
PHP has string interpolation, don't you prefer to use it? https://stackoverflow.com/questions/16790501/php-variable-interpolation-vs-concatenation